spec/lexers/gal_spec.rb

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*- #
+
```

Comment:
Can you please add '# frozen_string_literal: true'

Reply:
Hi, wow yeah this a pretty old PR, I had kind of forgotten about it.
I'm not sure, where should I add that line ?

Reply:
Just on the line after "# -*- coding: utf-8 -*- #". It freezes all String literals by default, which reduces memory usages, so you'll see it applied on all lexers now. E.g. if you look at lib/rouge/lexers/css:

```ruby
  # -*- coding: utf-8 -*- #
  # frozen_string_literal: true

  module Rouge
```